rho/authlistcommand.py

```diff
@@ -50,7 +50,7 @@ def _do_command(self):
         cred_list = vault.load_as_json(utilities.CREDENTIALS_PATH)
 
         for cred in cred_list:
-            if not cred.get('password') == '':
+            if cred.get('password') is not None:
```

Comment:
@elyezer Thoughts? I could certainly do the above, but then there would be no clarity if the user hadn't provided a password versus if they had.